icesat2_boreal_stac/constants.py

```diff
@@ -51,3 +47,264 @@ def matches_file(self, filename: str) -> bool:
     def required_assets(cls) -> Set[str]:
         """Returns set of all required asset types"""
         return {member.value for member in cls}
+
+
+VERSION = "v2.1"
+COLLECTION_ID_FORMAT = "icesat2-boreal-{version}-{variable}"
+
+RDS_MEDIA_TYPE = "application/x-rds"
```

Comment:
Why was this added? We should not be distributing any products in RDS format.

Reply:
It's leftover from when we were considering adding the Rds files as assets. I'll remove it.